runtime/lua/vscode/highlight.lua

```diff
@@ -0,0 +1,17 @@
+-- disable matchparen because we don't need it
+vim.g.loaded_matchparen = 1
```

Comment:
I wonder if this kind of stuff should go in a single file like `vscode/defaults.lua`, so that it is easy to document, easy to discover, and leaves the door open for a future "skip" flag if users want it for some reason.